include/zephyr/drivers/display.h

```diff
@@ -331,6 +347,10 @@ static inline int display_blanking_off(const struct device *dev)
 	struct display_driver_api *api =
 		(struct display_driver_api *)dev->api;
 
+	if (api->blanking_off) {
```

Comment:
```suggestion
	if (!api->blanking_off) {
```

Reply:
Outch. I switched from (!api->foo) to (api->foo == NULL) as "the last quick change" before commiting. Thank you for your keen eye.

Reply:
Missing:

```suggestion
	if (api->blanking_off == NULL) {
```

Reply:
Blanking off condition is not handled, the rest LGTM.